arrow/util.py

```diff
@@ -46,6 +52,20 @@ def is_timestamp(value):
         return False
 
 
+def is_ordinal(value):
+    """Check if value is a valid ordinal."""
+    if isinstance(value, bool):
+        return False
+    if not (isinstance(value, int) or isinstance(value, str)):
+        return False
+    try:
+        if int(value) not in range(MIN_ORDINAL, MAX_ORDINAL + 1):
+            return False
+    except ValueError:
+        return False
+    return True
```

Comment:
Hey @pgogri I'm not sure that we need this `util` function since the `datetime.fromordinal()` constructor we are wrapping should handle input checking.

Reply:
Hey @systemcatch thanks for looking into this. Yes we could just let datetime.fromordinal() handle input checking. It does value error for non-integer inputs. I guess the only difference, perhaps less significant, is it doesn't raise an exception for bools, (treats True as 1) should we try to check for this?

Reply:
Hmmm that's interesting it will accept `True` as input. I guess we have to leave the `bool` check in place, if we  just check for `int` that will fail since `bool` is a subclass of `int`.

Reply:
Yes true, I can modify the is_ordinal function to check for bools and integers and can remove the support for strings to keep this simple. How does that sound?